src/main/java/com/sharework/response/model/application/APIReceiptWorker.java

```diff
@@ -70,42 +62,37 @@ public Job(long id, String title, LocalDateTime startAt, LocalDateTime endAt, in
 		}
 	}
 
-	public static class JobTag {
-		@Column(name = "contents")
+	public static class RwJobTag {
 		public String contents;
 
-		public JobTag(String contents) {
+		public RwJobTag(String contents) {
 			this.contents = contents;
 		}
 	}
 
-	public static class Application {
+	public static class RwApplication {
 
-		@Id
-		@Column(name = "id")
 		public long id;
 
-		@Column(name = "isReview")
 		public Boolean isReview;
 
-		@ApiModelProperty(value = "job")
-		public Job job;
+		public RwJob job;
 
-		public Application(long id, Boolean isReview, Job job) {
+		public RwApplication(long id, Boolean isReview, RwJob job) {
 			this.id = id;
 			this.isReview = isReview;
 			this.job = job;
 		}
 	}
 
-	public static class User {
+	public static class RwUser {
```

Comment:
혹시 Rw 뜻이 ReceiptWorker 맞다면,
[오라클 가이드](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)에 따라 풀어쓰는게 더 나을 것 같아요

```suggestion
	public static class ReceiptWorkerUser {
```

위 제안을 적용하신다면, 나머지 이름명 수정은 제가 추가 커밋 넣을게요